app/grandchallenge/workstations/static/workstations/js/session-control.mjs

```diff
@@ -121,6 +123,9 @@ function setSpinner(element) {
 
 function removeSpinner(element) {
     const spinner = element.querySelector(".spinner-border");
+    if (spinner === null) {
+
+    }
```

Comment:
Leftover?

```suggestion
```

Reply:
Not totally, I got disrupted in person when working on this and completely forgot to finish the copy-paste =/. I noticed that running removeSpinner multiple times doesn't work, this is a fix.